src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkImageBrowser/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { Flex, ResponsiveBox } from "@artsy/palette"
+import { ArtworkImageBrowserFragmentContainer } from "Apps/Artwork/Components/ArtworkImageBrowser"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "relay-runtime"
+import { Media } from "Utils/Responsive"
+import { MyCollectionArtworkImageBrowser_artwork } from "__generated__/MyCollectionArtworkImageBrowser_artwork.graphql"
+import { MyCollectionArtworkNoImageComponent } from "./MyCollectionArtworkImageBrowser"
+
+interface MyCollectionArtworkImageBrowserProps {
+  artwork: MyCollectionArtworkImageBrowser_artwork
+}
+const MyCollectionArtworkImageBrowser: React.FC<MyCollectionArtworkImageBrowserProps> = ({
```

Comment:
And maybe we can rename this file to `MyCollectionArtworkImageBrowser`. I'm not quite sure if we still use `index.tsx` files in **Force** for components. In **Eigen** we try to avoid them ([see RFC](https://github.com/artsy/eigen/pull/6986)).